NuRadioReco/utilities/noise.py

```diff
@@ -508,15 +553,22 @@ def generate_noise(self, phasing_mode="slice", trigger_mode="binned_sum", debug=
             dt_triggering += time.process_time() - tstart
 
             if is_triggered:
-                triggered_bin = triggered_bin // 2  # the trace is cut in the downsampled version. Therefore, triggered bin is factor of two smaller. 
+                triggered_bin = triggered_bin // self.upsampling  # the trace is cut in the downsampled version. Therefore, triggered bin is factor of two smaller. 
```

Comment:
This is not introduced by this PR, but what happens if `trigger_mode == "stride"`? Wouldn't `triggered_bin` be undefined in that case?

Reply:
Fixed this :)